chat-plugins/info.js

```diff
@@ -1655,8 +1655,8 @@ exports.commands = {
 		if (!target) return this.parse('/help htmlbox');
 		target = this.canHTML(target);
 		if (!target) return;
-		if (!this.can('declare', null, room)) return;
 		if (!this.runBroadcast('!htmlbox')) return;
+		if (this.broadcasting && !this.can('declare', null, room)) return;
```

Comment:
Test `!htmlbox test` as a voiced user (a user with the `broadcast` but not the `declare` permission)